backend/db.js

```diff
@@ -47,44 +47,51 @@ export async function initializeDB() {
                 firstName VARCHAR(100),
                 middleName VARCHAR(10),
                 lastName VARCHAR(100),
-                birthDate VARCHAR(20),
                 maySave BOOLEAN,
+                birthDate TEXT,
                 creationDate DATETIME,
                 blacklisted BOOLEAN,
-                phoneNumber VARCHAR(20),
-                mailAddress VARCHAR(300)
+                phoneNumber TEXT,
+                mailAddress TEXT
             )
         `);
 
         db.run(`CREATE TABLE IF NOT EXISTS bookings (
                 id TEXT PRIMARY KEY,
-                customer_id TEXT,
-                duePayments INTEGER,
+                customer_id TEXT, 
                 startDate DATETIME,
                 endDate DATETIME,
-                amountChildren INTEGER,
-                amountAdults INTEGER,
-                expectedArrival TIME,
+                amountPeople INTEGER,
                 FOREIGN KEY (customer_id) REFERENCES customers (id)
             )
         `);
 
+        db.run(`CREATE TABLE IF NOT EXISTS payments (
+                booking_id TEXT,
+                amount INT,
+                status BOOLEAN,
+                note TEXT,
+                FOREIGN KEY (booking_id) REFERENCES bookings (id)
+        )`);
+
             // WHY IS IT CALLED card_uuid IT IS ALREADY IN A CARD RAAAAAAH
+            // `card_uuid` vervangen door `cardAddress`
 
         db.run(`CREATE TABLE IF NOT EXISTS cards (
                 id TEXT PRIMARY KEY NOT NULL,
-                card_uuid VARCHAR(16) NOT NULL,
+                cardAddress TEXT NOT NULL,
```

Comment:
dit staat niet in de ERD, ik geloof dat de ID gewoon de UUID van de fysieke card is